ntp-proto/src/identifiers.rs

```diff
@@ -20,7 +20,7 @@ impl ReferenceId {
         }
     }
 
-    pub(crate) fn from_int(value: u32) -> ReferenceId {
+    pub fn from_int(value: u32) -> ReferenceId {
```

Comment:
Given that we make the constants public, why does this then also need to be pub?